cmd/daprd/options/options.go

```diff
@@ -151,7 +150,7 @@ func (f stringSliceFlag) String() string {
 // Set the flag value.
 func (f *stringSliceFlag) Set(value string) error {
 	if value == "" {
-		return errors.New("value is empty")
+		return nil
```

Comment:
Previous behavior was a mistake.

In this case, the Dapr injector in 1.10 always includes a `--config` CLI flag, and if no config is passed, it's empty. This would have made daprd crash. So this change is required here to preserve forward-compatibility.